firefox_ui_tests/remote/security/test_unknown_issuer.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUnknownIssuer(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-unknownissuer.mozqa.com'
+
+    def test_unknown_issuer(self):
+        with self.marionette.using_context('content'):
+            # Go to a site that has a cert with an unknown issuer
+            self.assertRaises(MarionetteException, self.marionette.navigate, self.url)
+
+            # Wait for the DOM to receive events
+            time.sleep(1)
+
+            # Check the link in cert_domain_link
+            link = self.marionette.find_element(By.ID, 'cert_domain_link')
+            self.assertEquals(link.get_attribute('textContent'),
+                              'ssl-selfsigned-unknownissuer.mozqa.com')
+
+            # Verify the "Get Me Out Of Here!" and "Add Exception" buttons appear
+            self.assertIsNotNone(self.marionette.find_element(By.ID, 'getMeOutOfHereButton'))
+            self.assertIsNotNone(self.marionette.find_element(By.ID, 'exceptionDialogButton'))
```

Comment:
I think it may be better to check with `is_displayed` but that's more an enhancement, which we even didn't do in the Mozmill test. You can file a followup bug as a good first bug.